File: samples/server/petstore/jaxrs-cxf-cdi/src/main/java/org/openapitools/api/impl/StoreApiServiceImpl.java
package org.openapitools.api.impl;

import org.openapitools.api.*;
import org.openapitools.model.*;

import org.apache.cxf.jaxrs.ext.multipart.Attachment;

import java.util.Map;
import org.openapitools.model.Order;

import java.util.List;

import java.io.InputStream;

import jakarta.validation.constraints.*;
import jakarta.validation.Valid;

import jakarta.enterprise.context.RequestScoped;
import jakarta.ws.rs.core.Response;
import jakarta.ws.rs.core.SecurityContext;

@RequestScoped
@jakarta.annotation.Generated(value = "org.openapitools.codegen.languages.JavaJAXRSCXFCDIServerCodegen", comments = "Generator version: 7.5.0-SNAPSHOT")
public class StoreApiServiceImpl implements StoreApiService {
      @Override
      public Response deleteOrder(String orderId, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response getInventory(SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response getOrderById(Long orderId, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response placeOrder(Order order, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
}

File: samples/server/petstore/kotlin-server/javalin/src/main/kotlin/org/openapitools/server/apis/StoreApiServiceImpl.kt
package org.openapitools.server.apis

import org.openapitools.server.models.Order

class StoreApiServiceImpl : StoreApiService {

    override fun deleteOrder(orderId: String): Unit {
        TODO("Implement me")
    }

    override fun getInventory(): Map<String, Int> {
        TODO("Implement me")
    }

    override fun getOrderById(orderId: Long): Order {
        TODO("Implement me")
    }

    override fun placeOrder(order: Order): Order {
        TODO("Implement me")
    }
}
